### ISSUE TITLE
Add option to choose between calculating install size based on rom file or installation folder

### ISSUE THREAD

**Issue by Kimex123** [JosefNemec/Playnite#3093]
### Feature description

When  calculating install size of ps3 games playnite always calculates it based on the EBOOT.BIN file which causes the games to show up as being a couple og megabytes in size. Example, demon's souls is around 8 gigabytes but playnite the EBOOT.BIN file is only 26 megabytes

**Comment by ppvnf** [comment]
I agree. I am also having a similar problem for original Xbox games. The .xbe file is only 2mb but the folder is about 500mb

**Comment by rsa16** [comment]
Is there anyone currently working on this issue? If not I have some spare time on my hands and nothing to do so I'd like to work on this.

**Comment by Jeshibu** [comment]
[Games Size Calculator](https://playnite.link/addons.html#GamesSizeCalculator_97cc59db-3f80-4852-8bfc-a80304f9efe9) now has a fix for PS3 ROM sizes in the game right click menu.
